cray_product_catalog/catalog_update.py

```diff
@@ -65,6 +66,7 @@
 SET_ACTIVE_VERSION = bool(os.environ.get("SET_ACTIVE_VERSION"))
 REMOVE_ACTIVE_FIELD = bool(os.environ.get("REMOVE_ACTIVE_FIELD"))
 VALIDATE_SCHEMA = bool(os.environ.get("VALIDATE_SCHEMA"))
+SUB_CM_FIELDS_LIST = ['component_versions']
```

Comment:
i see a constant.py file, this can move there, if we are using as set , better to define as set.